solutions/aspnet/Makefile

```diff
@@ -25,10 +25,11 @@ export COMPlus_EnableDiagnostics=0
 export COMPlus_GCHeapHardLimit=0x8000000
 
 run: rootfs private.pem
-	test -f server.pid && kill -9 `cat server.pid` || true
-	timeout $(TIMEOUT) $(MYST_EXEC) $(OPTS) rootfs /app/webapp > server.output & echo $$! > server.pid
```

Comment:
The reason we recorded the wrong server pid is because of the `timeout` command. server.pid actually contains the pid of the `timeout` command, not the `myst` command.

Reply:
Thats what I thought, but having the kill.sh script is what we've been using as a standard for most of such applications, so I added that here - two birds with one stone